other-docs/guides/launching-a-site-on-altis.md

```diff
@@ -0,0 +1,181 @@
+
+## Planning
+
+Create a launch issue in the project repository. This issue should include:
+
+* Client name
+* Client business region
+* Client contact
+* Launch date
+* Feature freeze date
+* Code freeze date
+* Content freeze date (if migrating an existing site)
+* Team members needed for the deploy
+* Domains hosted on Altis
+* Domain(s) used for email
+* FROM address for each site in the network
+* GitHub repository
+* Deploy branch name per environment
+* A launch checklist containing pre-launch, launch day and post-launch tasks (examples can be found below)
+* Any additional notes or complexities for the project launch
+
+An [example checklist](#Example-launch-checklist) for your GitHub issue can be found below. This issue is primarily for _internal project use_ and compiling all the necessary information. 
+
+Once the GitHub issue is created, this information can then be copied into [a support ticket in the Altis Dashboard](https://docs.altis-dxp.com/guides/getting-help-with-altis/) to alert the Altis Support Team.
+
+## Example launch checklist
+
+```markdown
+# Goal
+
+<!-- A brief overview of the launch objective. Is it a migration? Is it a new site? -->
+
+## Pre-launch checklist
+<!-- To be filled out by the project team/lead. Not needed for Altis Support ticket. -->
+- [ ] Are we deploying the correct branch?
+- [ ] Has AWS SES been setup?
+- [ ] Is a content freeze active?
+- [ ] Is the client ready for DNS switches?
+- [ ] Site is ready for HTTPS?
+- [ ] Are any redirections needed?
+
+## Domains hosted on Altis
+<!-- List of domains that will be on the stack -->
+
+### Production
+- domain.com
+- domain2.com
+- etc.
+
+### Staging
+- domain-staging.altis.cloud
+- domain2-staging.altis.cloud
+- etc.
+
+### Development
+- domain-dev.altis.cloud
+- domain2-dev.altis.cloud
+- etc.
+
+**What domain(s) will email need to be sent from?**
+<!-- Enter as a list -->
+- [ ] noreply@domain.com
+- [ ] etc.
+
+**For each site in the network, what will be the FROM email?
+<!-- Enter as a list -->
+- [ ] **Network Admin:** noreply@domain.com
+- [ ] **domain.com:** some-email@domain.com
+- [ ] etc.
+
+## Environment
+* **GitHub repository:** https://github.com/humanmade/{projectname}
+* **GitHub repository deploy branches:**
+  * **Production:** `main`
+  * **Staging:** `staging`
+  * **Development:** `dev`
+  * **Altis integration branch:** `altis` <!-- If applicable -->
+* [ ] **Database exists in Altis Dashboard?**
+  * [ ] If not, provide a link to the database to import:
+* [ ] **Uploads exist in Altis Dashboard?**
+  * [ ] If not, provide a link to the uploads to import:  
+* **Who is providing the SSL certificate(s) (Altis or client)?:** 
+* **Who controls the DNS for the client (Altis or client)?:**
+* **Special requirements or considerations for ElasticSearch:**
```

Comment:
Let's drop this! We don't want to invite any requests for setting snowflakey environments